src/bridge/settings/openedx/version_matrix.py

```diff
@@ -13,15 +13,15 @@ class OpenLearningOpenEdxDeployment(Enum):
     mitx = DeploymentEnvRelease(
         deployment_name="mitx",
         env_release_map=[
-            EnvRelease("CI", OpenEdxSupportedRelease["quince"]),
+            EnvRelease("CI", OpenEdxSupportedRelease["redwood"]),
```

Comment:
You'll also need to add a definition for `redwood` to the `OpenEdxSupportedReleases` Enum in the `types.py` neighboring file.